src/vm/exceptionhandling.cpp

```diff
@@ -5156,7 +5156,7 @@ static BOOL IsIPinVirtualStub(PCODE f_IP)
     }
 
     VirtualCallStubManager::StubKind sk;
-    VirtualCallStubManager::FindStubManager(f_IP, &sk);
+    VirtualCallStubManager::FindStubManager(f_IP, &sk, false);
```

Comment:
A nit - could you please add comment with the argument name after the false to indicate what that false means for easier reading of the code? Like this:
```C++
VirtualCallStubManager::FindStubManager(f_IP, &sk, false /* usePredictStubKind */); 

```